src/unstract/adapters/x2text/helper.py

```diff
@@ -111,7 +117,7 @@ def make_request(
         }
         # Add api key only if present
         api_key = unstructured_adapter_config.get(UnstructuredHelper.API_KEY)
-        if api_key is not None and api_key != "":
+        if api_key:
```

Comment:
Will this also work for cases where api_key=""?

Reply:
Yes @gaya3-zipstack - it's redundant to check for empty string in Python and check for None. [This can help](https://stackoverflow.com/a/9573283/12864260)